esmvalcore/preprocessor/__init__.py

```diff
@@ -41,7 +41,7 @@
     mask_outside_range,
 )
 from ._multimodel import multi_model_statistics
-from ._other import clip
+from ._other import clip, add_cell_measure
```

Comment:
Maybe you could add a new module esmvalcore/preprocessor/_ancillary_variables.py and keep the code there? `other` is not very clear.